dgraph/cmd/debug/wal.go

```diff
@@ -122,55 +119,24 @@ func printBasic(store RaftStore) (uint64, uint64) {
 	return startIdx, lastIdx
 }
 
-func printRaft(db *badger.DB, store *raftmigrate.OldDiskStorage, groupId uint32) {
-	startIdx, lastIdx := printBasic(store)
-
-	commitTs := db.MaxVersion()
-	// In case we need to truncate raft entries.
-	batch := db.NewWriteBatchAt(commitTs)
-	defer batch.Cancel()
-	var numTruncates int
+func printRaft(store *raftwal.DiskStorage) {
+	isZero := store.Uint(raftwal.GroupId) == 0
 
+	// TODO: Fix the pending logic.
```

Comment:
This is still valid?

Reply:
I'm not sure what the pending logic is supposed to be. It was there in the earlier code too, so I've left this unchanged.

Reply:
Removed the comment, this seems to be handled in `printAlphaProposal`.